fix

```
➜  tui git:(main) bun tui.js 
error: Cannot find module '../lib/markdown-renderer.js' from '/home/dima/Projects/hives/polza-cli/tui/components/ChatView.jsx'

Bun v1.3.3 (Linux x64)
➜  tui git:(main) cd ..  
➜  polza-cli git:(main) npm install

up to date, audited 294 packages in 1s

91 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
➜  polza-cli git:(main) npm install

up to date, audited 294 packages in 1s

91 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
➜  polza-cli git:(main) bun tui/tui.js
error: Cannot find module '../lib/markdown-renderer.js' from '/home/dima/Projects/hives/polza-cli/tui/components/ChatView.jsx'

Bun v1.3.3 (Linux x64)

```

--- a/CLAUDE.md
+++ b/CLAUDE.md
@@ -135,13 +135,3 @@ Your forked repository: unidel2035/hives
 Original repository (upstream): judas-priest/hives
 
 Proceed.
-
----
-
-Issue to solve: undefined
-Your prepared branch: issue-91-185c4357
-Your prepared working directory: /tmp/gh-issue-solver-1764720481510
-Your forked repository: unidel2035/hives
-Original repository (upstream): judas-priest/hives
-
-Proceed.

